pkgs/applications/misc/pass-secret-service/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+with stdenv.lib;
+with python3.pkgs;
+
+buildPythonApplication rec {
+  pname = "pass_secret_service";
+  # Official release in PyPI is old, let's use latest from GitHub
+  version = "unstable-2020-04-12";
+
+  src = fetchFromGitHub {
+    owner = "mdellweg";
+    repo = pname;
+    rev = "f6fbca6ac3ccd16bfec407d845ed9257adf74dfa";
+    sha256 = "0rm4pbx1fiwds1v7f99khhh7x3inv9yniclwd95mrbgljk3cc6a4";
+  };
+
+  propagatedBuildInputs = [
+    click
+    cryptography
+    dbus-next
+    decorator
+    pypass
+    secretstorage
+  ];
+
+  # It seemed difficult to get the tests working..
```

Comment:
Can you also add why it is difficult?

Reply:
I think the issues were similar as with `pypass` that I described for your other question.

Reply:
Tests are now working.